tests/cypress/integration/actions_tasks_objects/case_29_settings_player_step.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('Settings "Player step"', () => {
+    const caseId = '29';
+    const countJumpStep = 3;
+    let startStep;
+
+    function changePlayerStep() {
+        cy.openSettings();
+        cy.get('.cvat-settings-modal').within(() => {
+            cy.contains('Player').click();
+            cy.get('.cvat-player-settings-step').within(() => {
+                cy.get('[role="spinbutton"]').clear().type(countJumpStep);
+            });
+        });
+        cy.closeSettings();
+    };
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Change player step ', () => {
+            changePlayerStep();
```

Comment:
What is reason to make a separated function here? The code is used only here as I can see.

Reply:
I thought this function will be use in several steps. It changed now.